R/sidebar.R

```diff
@@ -45,6 +45,8 @@
 #'   wrapped in a `<header>` element with class `sidebar-title`. You can also
 #'   provide a custom [htmltools::tag()] for the title element, in which case
 #'   you'll likely want to give this element `class = "sidebar-title"`.
+#' @param collapse_icon A [htmltools::tag()] child to use for the collapse icon.
+#'   Consider using a \pkg{bsicons} icon like `bsicons::bs_icon("sliders")`.
```

Comment:
We should probably mention here that the icon is decorative and we assume that it has `aria-hidden="true"`; it's placed in a button element with an appropriate label.

This does bring up that maybe we'd also want to include `collapse_label = "Toggle sidebar"` since we've currently hardcoded it in English.

Reply:
Good point about `aria-hidden="true"`. I might punt on `collapse_label` until we have a request for it.

Reply:
I'd consider the request for full screen toggle label customization to be a request for this label to be customizable. That said, I also think it's a pretty kludgy solution if authors would have to set `collapse_label` in each sidebar (some of which we make for them). So I'm okay waiting until we're ready to think about this more and provide a solid solution.